src-java/kilda-model/src/main/java/org/openkilda/model/HaSubFlow.java

```diff
@@ -116,6 +116,10 @@ public FlowEndpoint getEndpoint() {
         return new FlowEndpoint(getEndpointSwitchId(), getEndpointPort(), getEndpointVlan(), getEndpointInnerVlan());
     }
 
+    public boolean isOneSwitch() {
+        return getEndpointSwitchId().equals(getHaFlow().getSharedSwitchId());
+    }
+
```

Comment:
I have a similar function in #5176. So before merging check if this PR has been merged before to change this line.

https://github.com/telstra/open-kilda/blob/7785493d74162e80cf9aad7b8bfca380adbfe5b9/src-java/kilda-model/src/main/java/org/openkilda/model/HaSubFlow.java#L90